src/components/error-template/error-template.ts

```diff
@@ -58,6 +61,17 @@ export class MErrorTemplate extends ModulVue {
         return this.links.length > 0;
     }
 
+    public get iconUrl(): String {
+        return '../../assets/icons/svg/' + this.iconName + '.svg';
```

Comment:
Ca ne fonctionnera pas dans ena2-ui

tu dois le faire avec import et appende dans le dom.

Exemple: http://calebporzio.com/using-inline-svgs-in-vue-compoments/